src/auditwheel/main_addtag.py

```diff
@@ -30,7 +30,10 @@ def execute(args, p):
     try:
         wheel_abi = analyze_wheel_abi(args.WHEEL_FILE)
     except NonPlatformWheel:
-        logger.info("This does not look like a platform wheel")
+        logger.info(
+            "This does not look like a platform wheel"
+            ", no compiled extension file found in the wheel archive"
```

Comment:
Maybe something like the following: it has the precision about ELF file but also mention the extension bit as it might be more meaningful depending on one's knowledge of all of these details.

```suggestion
            ", no ELF executable or shared library file (including compiled Python C extension) found in the wheel archive"
```

PS: it probably won't pass linter validation as-is.

Reply:
Thanks for the input. I'll update the change